### DISCUSSION TITLE
Mod Items breaks your games screen

### DISCUSSION THREAD

**Discussion by GittySolo** [HbmMods/Hbm-s-Nuclear-Tech-GIT#729] [General]
Hey so I Downloaded the mod, launched it on 1.7.10 and upon join I'm greeted with a color screen that breaks my ability to see while the game is still running, but upon switching to a empty hand the screen fixes itself. Not sure why it does that but if I **hold anything that belongs to the mod** it breaks your screen into colors :/
![image](https://user-images.githubusercontent.com/112774628/188281830-b7fac9bd-f933-4e35-bfe2-d5c1092bfe52.png)
![image](https://user-images.githubusercontent.com/112774628/188281833-33056a89-be58-4a01-a597-73c711ebfb33.png)

**Comment by MCPlayer576** [comment]
Try reinstalling the mod or restart minecraft.

**Comment by BallOfEnergy1** [comment]
Do you have optifine or any sort of rendering improvement mod? HBM NTM is notorious for bugging out when combined with (most) any mod that changes the rendering.

**Comment by HbmMods** [comment]
A general issue with 1.7.10, there's a few solutions listed on #985
